CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Added `enableAutomaticViewTracking` config for automatic track views
 - Added `automaticViewTrackingExclusionList` config for automatic view tracking exclusion list
 - Added `globalViewSegmentation` config to add set global view segmentation.
+- Added `enrollABOnRCDownload` config for auto enrolling AB when downloading values.
```

Comment:
I guess we might need to mention the downloading 'RC values' , sth likethis mb?:
Added `enrollABOnRCDownload` config method to auto enroll users to AB tests when downloading RC values.